internal/services/redhatopenshift/redhat_openshift_cluster_resource.go

```diff
@@ -193,6 +194,12 @@ func (r RedHatOpenShiftCluster) Arguments() map[string]*pluginsdk.Schema {
 							false,
 						),
 					},
+					"preconfigured_nsg_enabled": {
```

Comment:
To match other fields in the provider, can we spell this out as `network_security_group` here:

```suggestion
					"preconfigured_network_security_group_enabled": {
```

Reply:
Apologies, I missed the fact that you made this change and I simply needed to commit.  Regardless, I fixed in commit becf28602323ab08a37def639367eb0975a74629